opta.spec

```diff
@@ -4,7 +4,37 @@ from PyInstaller.utils.hooks import collect_data_files
 
 block_cipher = None
 
-
+himps = [
```

Comment:
can we sort this list for maintenance?
also it can be broken into 3 lists for each cloud providers

Reply:
Can we instead have a function which generates this list

Reply:
@RemyDeWolf @NitinAgg great suggestions! Addressed both  in new commit.